### DISCUSSION TITLE
Multi tenant support

### DISCUSSION THREAD

**Discussion by khqnn** [reactioncommerce/reaction#6843] [Q&A] [answered]
Do reaction provides full support for multitenancy with which multiple tenants/stores sharing the same database and having isolation of data. Also to restrict all endpoints like products etc needs the tenant id to access and modify their own data.

**Comment by brent-hoover** [comment, marked as answer]
Yes, I believe some people are actually using this in production. Multi-tenant support has been built in from the get-go in the form of "shops".
